src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {` `}
+      {person.name}
+    </h2>
+    {person.age !== undefined && (
+    <p className="Person__age">
+      <span>{`I am ${person.age}`}</span>
+    </p>
+    )}
+    <p className="Person__partner">
+      {(person.isMarried && person.sex === 'm') ? <span>{`${person.partnerName} is my wife`}</span> : (
+        ''
+      )}
+      {(person.isMarried && person.sex === 'f') ? <span>{`${person.partnerName} is my husband`}</span> : (
+        ''
+      )}
+      {(!person.isMarried) ? <span>I am not married</span> : (
+        ''
+      )}
```

Comment:
let's refactor it
you can make two ternary before return jsx and deside which text will be rendered.
it will be shorter and more readable.

```
const partner = person.sex === 'm' ? 'wife' : 'husband';
const personPartnerText =  person.isMarried ? `${person.partnerName} is my ${partner}` : 'I am not married'
```

```suggestion
   {personPartnerText}
```